UI/src/less/site/modals.less

```diff
@@ -65,6 +65,68 @@
   background: rgba(red(@gray-darkest), green(@gray-darkest), blue(@gray-darkest), 0.6);
 }
 
+.modal-overlay.confirm-modal {
+
+  .selected {
+    font-weight: bold;
+    font-size: 13px;
+  }
+
+  .bold {
+    font-weight: bold;
+  }
+
```

Comment:
.bold and .selected should be inside .notification, **.modal-overlay.confirm-modal** is used in more places together